commands/credentials.js

```diff
@@ -51,11 +52,36 @@ function * run (context, heroku) {
         return renderAttachment(attachment, app, isLast)
       })
 
-      return [cred].concat(attLines).join('\n')
+      let rotationLines = []
+      let credentialStore = credentials.filter(a => a.name === cred)[0]
+      if (credentialStore.state === 'rotating') {
+        let formatted = credentialStore.credentials.map(function (credential, idx) {
+          return {
+            'user': credential.user,
+            'state': credential.state,
+            'connections': credential.connections
+          }
+        })
+        let connectionInformationAvailable = formatted.some(function (c) { return c.connections })
+        if (connectionInformationAvailable) {
+          let prefix = '       '
+          rotationLines.push(`${prefix}Usernames currently active for this credential:`)
+          cli.table(formatted, {
+            printHeader: false,
+            printLine: function (line) { rotationLines.push(line) },
+            columns: [
+              {key: 'user', format: (v, r) => `${prefix}${v}`},
+              {key: 'state', format: (v, r) => (v === 'revoking') ? 'waiting for no connections to be revoked' : v},
```

Comment:
yes :-S i want to express the fact that it is in the rotation process at the point where we are waiting for the credential to have no connections before revoking it. any copy suggestion would be super welcome ❤️